tests/trello/mock_trello_api_json_responses.py

```diff
@@ -0,0 +1,286 @@
+mock_trello_api_json_responses = {
```

Comment:
In this project the fixtures are in JSON files rather than python objects. Cf. https://github.com/ToucanToco/toucan-connectors/tree/master/tests/micro_strategy/fixtures

Reply:
Done !